bd-logger/src/async_log_buffer.rs

```diff
@@ -680,6 +717,7 @@ impl<R: LogReplay + Send + 'static> AsyncLogBuffer<R> {
     session_id: String,
     occurred_at: time::OffsetDateTime,
   ) -> anyhow::Result<()> {
+    // TODO(mattklein123): Should we support injected logs for internal logs?
```

Comment:
I think this just puts logs on the async queue so it should just work?

Reply:
Let me double check I think this is direct and not via the queue so it needs to actually handle the injected logs returned.